dotnetv3/S3/scenarios/TransferUtilityBasics/TransferUtilityBasicsTests/TransferMethodsTests.cs

```diff
@@ -1,65 +1,87 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Amazon;
-using Amazon.S3;
-using Amazon.S3.Transfer;
-using TransferUtilityBasics;
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
 
 namespace TransferUtilityBasics.Tests
 {
-    [TestClass()]
     public class TransferMethodsTests
     {
-        IAmazonS3 client;
-        TransferUtility transferUtil;
-        string LocalPath = $"{Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)}\\TransferFolderTest";
-        string bucketName;
+        private readonly IConfiguration _configuration;
 
-        TransferMethodsTests()
+        readonly IAmazonS3 _client;
+        readonly TransferUtility transferUtil;
+        private readonly string _localPath = $"{Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)}\\TransferFolderTest";
+
+        public TransferMethodsTests()
         {
-            client = new AmazonS3Client();
-            transferUtil = new TransferUtility();
-            bucketName = "doc-example-bucket1-test";
+            _client = new AmazonS3Client();
+            transferUtil = new TransferUtility(_client);
+
+            _client = new AmazonS3Client();
+            _configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("testsettings.json") // Load test settings from JSON file.
+                .AddJsonFile("testsettings.local.json",
+                    true) // Optionally load local settings.
+                .Build();
         }
 
-        [TestMethod()]
+        [Fact()]
         public async Task DownloadSingleFileAsyncTest()
         {
-            var keyName = "FileToDownload.docx";
-            var success = await TransferMethods.DownloadSingleFileAsync(transferUtil, bucketName, keyName, LocalPath);
-            Assert.IsTrue(success, $"Couldn't download {keyName}.");
+            var keyName = _configuration["FileToDownload"];
+            var success = await TransferMethods.DownloadSingleFileAsync(
+                transferUtil,
+                _configuration["BucketName"],
+                _configuration["FileToDownload"],
```

Comment:
You could reuse the local variable here, keyName.

Reply:
fixed